features/git-town/configuration/configuration_reset.feature

```diff
@@ -0,0 +1,36 @@
+Feature: resetting the configuration
+
+  As a user wishing to remove all Git Town settings from a repository
+  I want to be able to cleanly reset the Git Town configuration with a single command
+  So that I can continue working on actual work and stay productive
```

Comment:
Perhaps the user doesn't wish to use Git Town anymore, and doesn't want any remnants of it polluting the repo?

The reason I added this was actually to make it easier to test my changes regarding initial configuration.